### ISSUE TITLE
AsyncAPI 3.0.0 preview doesn't work when referencing local avro files

### ISSUE THREAD

**Issue by JuanPalacios** [asyncapi/vs-asyncapi-preview#238]
### Describe the bug.

AsyncAPI 3.0.0 preview doesn't work when referencing local avro files

### Expected behavior

AsyncAPI 3.0.0 preview works correctly when referencing local avro files. This functionality with AsyncAPI version 2.6.0 and version 0.5.3 works correctly.

### Screenshots

The preview is not displayed:
![image](https://github.com/user-attachments/assets/9b7648fd-b35d-4769-b03a-fbde116baf9d)

The only error shown in AsyncAPI studio is that it does not support reference to local files, so we understand that the definition is correct.
![image](https://github.com/user-attachments/assets/8555b2c3-ae55-4734-aa32-3a0721045af2)

### How to Reproduce

1. We have AsyncAPI 2.6.0 working correctly with plugin version 0.5.3 and references to local avros.

2. I have created an AsyncAPI 3.0.0 without local avros references and it works correctly with plugin version 0.6.3.

3. When I add a reference to a local avro the screen goes blank and we can't see the AsyncAPI. It prevents us from continuing with our upgrade process as we cannot see the AsyncAPI at all.

### 🥦 Browser

None

### 👀 Have you checked for similar open issues?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Contributing Guidelines?

- [X] I have read the [Contributing Guidelines](https://github.com/asyncapi/.github/blob/master/CONTRIBUTING.md)

### Are you willing to work on this issue ?

No, someone else can work on it

**Comment by ivangsa** [comment]
Hi @JuanPalacios 

thanks for taking the time to report this.
do you mind attaching a zip with the folder structure that is making this fail?

thanks again

**Comment by JuanPalacios** [comment]
Here you are!
[events-dictionary.zip](https://github.com/user-attachments/files/17278502/events-dictionary.zip)

**Comment by ivangsa** [comment]
hi @JuanPalacios 

it doesn't find the file because the url is wrong: 

![image](https://github.com/user-attachments/assets/58156ea6-7b0d-4d5c-937b-8386c80f5b86)

but even with the right url it doesn't seem to open the preview:

```yml
$ref: '../../../../avro/src/main/resources/pre/com/masmovil/commons/avro/provision/broadband/v1/activation/activationstarted.avsc'
```
I need to investigate

**Comment by JuanPalacios** [comment]
Yes, you are right.  I was trying other paths, but the rigth one is the one you write.

**Comment by ivangsa** [comment]
I found why it doesnt render:

this is the proper format to reference avro files

```yml
      payload:
        schemaFormat: 'application/vnd.apache.avro;version=1.9.0'
        schema:
          $ref: '../../../../avro/src/main/resources/pre/com/masmovil/commons/avro/provision/broadband/v1/activation/activationstarted.avsc'
```

but this aliases line in the .avsc is also somehow breaking the parser

`  // "aliases" : [ "topic.activationStarted", "compatibility.full_transitive" ]`

if you comment that line and use the format above you will get the API rendered.

I'm not sure if is not a valida avro schema or the parser does not support aliases

**Comment by JuanPalacios** [comment]
wow, very fast, in my opinion the aliases are totally right because we have published this avros in our confluent schema registry, see the image below:

![image](https://github.com/user-attachments/assets/8341b5d2-069d-4093-9a12-65484eaa370b)

**Comment by ivangsa** [comment]
worth rising an issue/question with the parser project, bc I don't even get the error exception for this, but if I inline the avro inside the asyncapi.yml I get a linting error:

![image](https://github.com/user-attachments/assets/f4a7554c-d95a-4ffb-a96b-ae6e907c70c4)

**Comment by carlreid** [comment]
> worth rising an issue/question with the parser project

Was an issue opened for this? Struggling to find it if so. I also have encountered this problem.

I guess the regex pattern needs updating to allow support for `.`  - at least for me, it's because the alias is trying to reference a fully qualified type using it's namespace which contains `.`. If I replace `.` for `_` it works fine, which makes sense as it passes the regex pattern.
